test-network/prometheus-grafana/README.md

```diff
@@ -0,0 +1,47 @@
+# Description
+
+This sample provides an environment to display and capture metrics from the test-network in real time. It consists of a docker-compose file that starts a prometheus and grafana server setup configured to collect and display metrics for the test network.
+
+# Requirements
+
+This sample is meant to be used on **linux** in order to fully benefit from its capabilities. However, it can be used and adapted on other systems with some additional work and tweaks.
+
+You will need to have installed **docker-compose with version 1.29 or above**.
+
+# How to use
+
+1. Bring up the test-network 
+2. Bring up the Prometheus/Grafana network --> **docker-compose up -d**
+3. Log in: type “< address >:3000” on your web browser -> username=“admin”, password=“admin” 
```

Comment:
add in the localhost rather than address///

Reply:
maybe worth noting that you'll need to change the password when you first log in as well